test/evergreen.yml

```diff
@@ -2386,6 +2386,24 @@ tasks:
           times: 1       # No of times to run the loop
           no_of_procs: 10 # No of processes to run in the background
 
+  - name: performance-test
+    tags: ["performance-test-1"]
```

Comment:
Why do we need this tag? Tags are usually used to group tasks together, making calling them easier (by using tag selector feature).

Reply:
I was thinking that if we keep creating stress testings, creating tags right now could prepare that.

Reply:
Got it. I think I prefer the more lazy approach - applying tags whenever there's a real need :)